projects/frontend/shared-components/gui/projects/shared/src/lib/commons/ngx-utils/simple-translate-service/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
I believe once the l10n support is removed from the components, this service will be redundant and can be wiped out as well.

Reply:
I'm not messing with l10 support in this PR. It requires me to change too much code as lots of components actually use this service. We can address it as a separate issue.